resources/build/tests/builder.inc.test.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 ## START STANDARD BUILD SCRIPT INCLUDE
```

Comment:
This script has lost its `+x` attr -- and because it has `.inc` in the name, isn't triggering any yammering from the pre-commit hook 😆